app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/ImageZoom.java

```diff
@@ -0,0 +1,114 @@
+package org.pesmypetcare.mypetcare.activities.fragments;
+
+import android.content.Intent;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import org.pesmypetcare.mypetcare.R;
+import org.pesmypetcare.mypetcare.activities.views.CircularImageView;
+import org.pesmypetcare.mypetcare.databinding.FragmentImageZoomBinding;
+
+import java.util.Objects;
+
+public class ImageZoom extends Fragment {
+    private static final String[] IMAGE_MIME_TYPES = {"image/jpeg", "image/png"};
+    private static final int GALLERY_ZOOM_REQUEST_CODE = 0;
+    private static final float RADIUS = 1000.0f;
+
+    private FragmentImageZoomBinding binding;
+    private static Drawable drawable;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5223641)